app/models/licensee.rb

```diff
@@ -0,0 +1,2 @@
+class Licensee < ActiveRecord::Base
+end
```

Comment:
Out of curiosity, did you actually run `rake db:migrate` and confirm whether your migrations were set up correctly?